containers/podman-desktop
# chore: reusable base page

### What does this PR do?

Provides a single base layout page that has enough options to support the current FormPage and DetailsPages.

This is enough that both Help and Troubleshooting pages can use this new layout instead of FormPage. This means they need to handle the close directly (maybe it's worth another page in the middle to handle this for both?), but it means we can remove the showBreadcrumb option from FormPage to start to clean it up.

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

First part of #7757. (NavPage, SettingsPage, or merging in any other pages is not attempted here)

Fixes #7739.

### How to test this PR?

Check several form pages, details pages, troubleshooting, and help for regressions/change, should be none.
